src/kl_exchange.jl

```diff
@@ -16,6 +16,78 @@ function d_criterion(model_matrix;
         (1 / size(model_matrix, 2))
 end
 
+
+"""
+$(TYPEDSIGNATURES)
+
+Criterion of A-optimality.
```

Comment:
Could you add a short description and perhaps a formula too like I did for the D-criterion?

Reply:
It would be nice to have examples or tests to be able to check the designs produced after KL-Exchange make sense, that is, that they are the expected results from the criteria used

Reply:
Hi! Some description added. 

Papers used:

https://v8doc.sas.com/sashtml/qc/chap24/sect30.htm

https://www.researchgate.net/publication/267494190_Optimality_Criteria_for_Measurement_Poses_Selection_in_Calibration_of_Robot_Stiffness_Parameters

https://escholarship.org/content/qt2bz9c0zr/qt2bz9c0zr_noSplash_527ff2e3214c37a6d5d847901f464c18.pdf

https://ncss-wpengine.netdna-ssl.com/wp-content/themes/ncss/pdf/Procedures/NCSS/D-Optimal_Designs.pdf

Reply:
Thanks, it's looking pretty good, I'll merge now.